ports/servoshell/webview.rs

```diff
@@ -205,16 +210,32 @@ where
                         }
                     },
                     EventType::Connected => {
+                        let len = self.haptic_effects.len();
+                        self.haptic_effects.resize_with(len + 1, Default::default);
```

Comment:
The intention was to add a new slot in the haptic effects vec for any effects that will play on that gamepad, not sure why I did it like this instead of just appending None though. Have adjusted this